packages/jest-cli/src/TestRunner.js

```diff
@@ -59,7 +59,7 @@ class TestRunner {
     options: Options,
   ) {
     this._config = config;
-    this._dispatcher = new ReporterDispatcher();
+    this._dispatcher = new ReporterDispatcher({hasteContext: hasteMap});
```

Comment:
i did thin and then rolled back :) because the actual map is in hasteContext and not haste map. there is a lot of naming mismatches like `hastemap->hastecontext` but i didn't touch them and i supposed we'll fix them later :)